### PR TITLE
remove unnecessary remote.Locate()

### PR DESCRIPTION
**Pull request by itscaro**

### CODE REVIEW

**Comment by yxxhero** [comment]
@itscaro fixed unittest.

**Comment by itscaro** [comment]
https://github.com/helmfile/helmfile/actions/runs/3662974114/jobs/6192445236#step:7:2153
image

https://github.com/helmfile/helmfile/actions/runs/3663336337/jobs/6193050525#step:7:2153
image

@yxxhero there is still an strange behavior

**Comment by yxxhero** [comment]
yeah. wait for me to resolve this.

**Comment by yxxhero** [comment]
@itscaro fixed. is it ready to review?

**Comment by yxxhero** [comment]
@itscaro I think that need some tests for covering your changes.

**Comment by itscaro** [comment]
@yxxhero i will try to add some tests next week. This removal is because after the locate() call, there is already resolveFile() which does the some thing.

**Comment by itscaro** [comment]
@yxxhero i am busy at this end of the year, I can only handle this PR near mid-january 2023 :(

**Comment by yxxhero** [comment]
@itscaro  any updates?

**Comment by yxxhero** [comment]
@itscaro ping

File: pkg/state/envvals_loader.go [hunk 1]
```diff
@@ -42,13 +42,7 @@ func (ld *EnvironmentValuesLoader) LoadEnvironmentValues(missingFileHandler *str
 
 		switch strOrMap := entry.(type) {
 		case string:
-			urlOrPath := strOrMap
```
Comment (yxxhero) [inline pkg/state/envvals_loader.go:45]: @itscaro would you mind adding some explanation for this change?

Comment (itscaro) [reply pkg/state/envvals_loader.go:45]: the function ld.storage.resolveFile() resolves 
itself url or path so we do not  need do the resolution part here.

https://github.com/helmfile/helmfile/pull/565/files/877560c09cf56305adb6bcfdc612a1b760c61026#diff-9971a72e8dfdc9ad176f7a241e51ef028950bfda8ab89de9b67a5e41ef072700R45

**Review by yxxhero — APPROVED** [review]

**Pull request merged**